components/builder-web/app/actions/gitHub.ts

```diff
@@ -223,14 +223,20 @@ function resetGitHubRepos() {
 }
 
 // Return up to two trailing segments of the current hostname
-// for purposes of setting the cookie domain.
+// for purposes of setting the cookie domain unless the domain
+// is an IP address.
 function cookieDomain() {
     let delim = ".";
+    let tld = currentHostname().split(delim).pop();
 
-    return currentHostname()
+    if (Number(tld) === NaN) {
```

Comment:
Knowing practically nothing about TypeScript, is this the idiomatic way to ask if a variable is a number (or not, as the case may be)?

Reply:
sounds like we probably know the same amount of typescript. Thats what I stole from stackoverflow, but I'll defer to our TS experts.

Reply:
oh that was actually wrong. this seems better.